internal/pkg/local/manager.go

```diff
@@ -73,6 +73,13 @@ func (u *UnitOfWork) SkipNotFoundErr() {
 	u.skipNotFoundErr = true
 }
 
+func (u *UnitOfWork) LoadedObjects() []model.Object {
```

Comment:
Z `UnitOfWork` potrebujem, po tom co dobehne, ... zistit vsetky objekty, ktore nacita.
Preto pridana tato metoda, + premenovane `new -> loaded`,... aby to bolo vystiznejsie.